filters/diag/diag.go

```diff
@@ -417,3 +445,58 @@ func (t *throttle) Response(ctx filters.FilterContext) {
 	rsp := ctx.Response()
 	rsp.Body = t.goThrottle(rsp.Body, true)
 }
+
+func (j *jitter) Name() string {
+	switch j.typ {
+	case normalDist:
+		return "jitterNormal"
+	case uniformDist:
+		return "jitterUniform"
+	}
+	return "unknown"
+}
+
+func (j *jitter) CreateFilter(args []interface{}) (filters.Filter, error) {
+	var (
+		mean  time.Duration
+		delta time.Duration
+		err   error
+	)
+
+	if len(args) != 2 {
+		return nil, filters.ErrInvalidFilterParameters
+	}
+	if mean, err = parseDuration(args[0]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration mean %v: %w", args[0], err)
+	}
+
+	if delta, err = parseDuration(args[1]); err != nil {
+		return nil, fmt.Errorf("failed to parse duration delta %v: %w", args[1], err)
+	}
+
+	return &jitter{
+		typ:   j.typ,
+		mean:  mean,
+		delta: delta,
+	}, nil
+}
+
+func (j *jitter) Request(filters.FilterContext) {
+	d := j.mean
+
+	switch j.typ {
+	case uniformDist:
+		/* #nosec */
+		r := 2*rand.Float64() - 1   // +/- sizing
+		f := r * float64(2*j.delta) // resize to specified delta
```

Comment:
For
```go
r := 2*rand.Float64() - 1
```
we do not need to multiply delta, i.e. it should be 
```go
f := r * float64(j.delta)
```

Reply:
ok thanks, fixed